src/components/contacts/ContactRow.js

```diff
@@ -159,23 +149,6 @@ const ContactRow = (
 
   const imageAvatar = profilesEnabled ? ensAvatar?.imageUrl : image;
```

Comment:
i'm wondering if we should default to image here if profilesEnabled? 
what happens when ensAvatar?.imageUrl is undefined but image is defined?

Reply:
nice catch!